### PR TITLE
Writing and more

### PR DESCRIPTION
**Pull request by tshuck**
This PR:
  - Removes styled-components in favor of emotion, since rebass seems to favor that natively.
  - Adds a favicon
  - Does a bit of styling clean-up
  - Adds the first _existing in the world_ writing "On Goals and Happiness"

Any editing or revision feedback on structure or content is welcome from The Internet, especially on the new piece, is gratefully welcomed.

### CODE REVIEW

**Comment by tshuck** [comment]
@DLavin23 You know, if you're just dying for something to do on your Tuesday off.

File: src/content/writing/2019-10-09-on-goals-and-happiness.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+#### A Page
+
+The particular idea of who I am goes back only ten or twelve years. Any further past and I have wide impressions: an easy ten-year-old desire to be, so absurdly generally, a _scientist_; the sharp sense of freedom while driving alone late at night with a new license<Footnote id="foot-1" href="#1">1</Footnote>. I can see the beginnings of myself, but the patterns and systems of my brain, my sense of being, do not feel coherently connected to who I am now. At eighteen I picked up the first thread of my personality that I can follow through to today -- when asked what I wanted to do, or to be, I would say that I only wanted to be happy. That was my goal; I am not goal oriented.
```
Comment (DLavin23) [inline src/content/writing/2019-10-09-on-goals-and-happiness.mdx]: > At eighteen I picked up the first thread of my personality that I can follow through to today -- when asked what I wanted to do, or to be, I would say that I only wanted to be happy. That was my goal; I am not goal oriented.

I love this sentence, especially the "That was my goal; I am not goal oriented." Very on brand, stylistically.

File: src/content/writing/2019-10-09-on-goals-and-happiness.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+I carried this goal, _happiness_, through school<Footnote id="foot-2" href="#2">2</Footnote> and along through the beginnings of my adulthood and career. I'll say at the outset, here, that happiness is a good direction and an awful goal. I'll say also that I was more earnest than smugly anti-establishment; I inherited a certain casual _everything works out_ attitude from my mom that I intellectualized to the point of being deeply unhelpful. It's true, too, that everything does work out, one way or another, and that I've been lucky. I come from a background of white- and male-privilege. I'm grateful for what I have -- what I've been given and what I've earned.
```
Comment (DLavin23) [inline src/content/writing/2019-10-09-on-goals-and-happiness.mdx]: Whenever me and my sisters would complain about something being fair or unfair, my mom would always tell us that "everything evens out when you're 21."

File: src/content/writing/2019-10-09-on-goals-and-happiness.mdx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+Being happy, not letting things bother me, being _okay_, was a foundational part of my identity. This is aspirational and it's not entirely workable. In practice it looked like this: spending an enormous amount of mental energy to dismiss my feelings when things were _not okay_. It meant ignoring problems, rationalizing, and overindulging in patience. It meant, especially, avoiding emotional conflict.
+
+I did not often step into my life when it was hard and when I needed to. I wrapped discontent, sadness, and irritation in an intellectual mantra: I don't need to feel this. It will be okay. This was my process for existing; I have a habit of over-thinking and under-feeling<Footnote id="foot-3" href="#3">3</Footnote>. In my mid-twenties this slowly stopped working, and I descended into a semi-functional madness. My sense of identity was rapidly crumbling. I reached a breaking point, and I felt that I had broken. I did not wish to die, but in the morning I would pull the covers over my head, and hide, and think about not existing.
```
Comment (DLavin23) [inline src/content/writing/2019-10-09-on-goals-and-happiness.mdx]: I'm sorry you went through that. I'm glad you've made it through to the other-side, hopefully stronger and better off.

File: src/content/writing/2019-10-09-on-goals-and-happiness.mdx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+I do not feel broken today. In my late twenties I began to be reassembled<Footnote id="foot-4" href="#4">4</Footnote>. This process continues. Here is what I learned: **happiness itself is not actionable**<Footnote id="foot-5" href="#5">5</Footnote>. Being okay is a direction to travel and, importantly, is an _outcome_ of the structure of my life. Feeling bad is a deeply helpful signal to understand what I need to change and how I need to grow; where I need to assert myself, and how to step into my life.
```
Comment (DLavin23) [inline src/content/writing/2019-10-09-on-goals-and-happiness.mdx]: insert transformer being assembled or something?

File: src/content/writing/2019-10-09-on-goals-and-happiness.mdx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+I've kept the pieces that I like: optimism, patience, kindness. I've added to my process for existing. It's still very intellectual<Footnote id="foot-6" href="#6">6</Footnote>, and I do a lot of work to understand and incorporate how I feel. Sometimes it's very easy: I'm irritable when I'm hungry. Sometimes it's enough to recognize feeling uneasy because life is uncertain. Sometimes I have to step in and have a conversation with someone that is wildly uncomfortable. I have a mental checklist that, so importantly, starts when I actively engage with the thought or feeling that conflicts with _being happy_<Footnote id="foot-7" href="#7">7</Footnote>.
+
+This optimistic thread of my personality began partially as a naive defense mechanism to avoid conflict, and I've had to learn a lot about the work that must be done for it to be a useful part of myself. I'm still not particularly goal-oriented. Having goals, and growing, are both skills that I develop slowly. I am happy, generally, and importantly I know what to do when I'm not. I don't know that this is helpful advice, only that it's been a gradual realization.
```
Comment (DLavin23) [inline src/content/writing/2019-10-09-on-goals-and-happiness.mdx]: > I am happy, generally, and importantly I know what to do when I'm not. I don't know that this is helpful advice, only that it's been a gradual realization.

👍🏻👍🏻

File: src/content/writing/2019-10-09-on-goals-and-happiness.mdx [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+---
+
+import { Box, Text, Link } from 'rebass'
+import { Footnote } from '../../components'
+
+<Box fontSize={3}>
+
+#### A Sentence
+
+Happiness is not a goal; it is an outcome of engaging with both the positive and negative aspects of my life.
```
[the remark this answers is not in the document]
Comment (tshuck) [reply src/content/writing/2019-10-09-on-goals-and-happiness.mdx]: I prefer to think I'm channeling Jason Fried's prairie restoration rituals, but...

**Comment by tshuck** [comment]
@DLavin23 Thank you for your kind words, your empathy and your suggestions.  I added some foxes to break up the text.  And an addendum that's questionably helpful.

**Pull request merged**